README.md

```diff
@@ -65,6 +65,11 @@ Des blagues et des animaux. Proposer la vôtre si elle me faire rire, je l'inclu
 
 ---
 
+**Q:** Quel animal est enterré dans une pyramide ?
+
+**A:** Toucankamon
```

Comment:
Replace the A by a R.

Reply:
@aaiaueil When action done. Please write "Done." Below the sections to revise for an efficient review.